### DISCUSSION TITLE
tsserver not showing `navigator.gpu` and any of its members.

### DISCUSSION THREAD

**Discussion by SlinkierElm5611** [typescript-language-server/typescript-language-server#762] [Q&A] [answered]
Hi all, I am working on some work related to wgpu and have been having some issues as it seems that tsserver does not show any completions for it. Is support for this planned?

**Comment by rchl** [comment, marked as answer]
It might not include support for features that are not standardized yet. You probably need to find and use third party types in the meantime (it seems like there is something at https://github.com/gpuweb/types).
